docs/3.api/1.components/4.nuxt-link.md

```diff
@@ -25,6 +25,23 @@ In this example, we use `<NuxtLink>` component to link to another page of the ap
 </template>
 ```
 
+### Handling 404s
+
+When using `<NuxtLink>` for `/public` directory files or when pointing to a different app on the same domain, you should use the `external` prop.
```

Comment:
I couldn't think of any other valid use cases for manually using `external`, let me know if you think of anything